pyodata/v2/model.py

```diff
@@ -175,15 +175,14 @@ def parse(value):
 
 
 class Types:
-    """Repository of all available OData types
+    """Repository of all available OData V2 primitive types + their Collection variants
 
        Since each type has instance of appropriate type, this
        repository acts as central storage for all instances. The
        rule is: don't create any type instances if not necessary,
        always reuse existing instances if possible
     """
 
-    # dictionary of all registered types (primitive, complex and collection variants)
```

Comment:
Maybe this was intent in the start, but now is IMHO misleading.

Reply:
Reviewing your own changes? Better to put your comments to commit message then.